ra/ra.go

```diff
@@ -566,108 +566,115 @@ func (ra *RegistrationAuthorityImpl) checkNewOrdersPerAccountLimit(ctx context.C
 
 // NewAuthorization constructs a new Authz from a request. Values (domains) in
 // request.Identifier will be lowercased before storage.
-func (ra *RegistrationAuthorityImpl) NewAuthorization(ctx context.Context, request core.Authorization, regID int64) (core.Authorization, error) {
-	identifier := request.Identifier
```

Comment:
ugh, I hadn't even noticed that the old version completely shadows the `identifier` package with this name. Thanks for changing it!